src/main/scala/ch/openolitor/stammdaten/repositories/StammdatenRepositoryQueries.scala

```diff
@@ -593,9 +593,15 @@ trait StammdatenRepositoryQueries extends LazyLogging with StammdatenDBMappings
       select
         .from(lieferungMapping as lieferung)
         .leftJoin(vertriebMapping as vertrieb).on(lieferung.vertriebId, vertrieb.id)
+        .leftJoin(abwesenheitMapping as abwesenheit).on(abwesenheit.lieferungId, lieferung.id)
         .where.eq(lieferung.abotypId, abotypId).and.eq(vertrieb.abotypId, abotypId).and.eq(lieferung.vertriebId, vertriebId).and.isNull(lieferung.lieferplanungId)
         .orderBy(lieferung.datum)
-    }.map(lieferungMapping(lieferung)).list
+    }.one(lieferungMapping(lieferung))
+      .toMany(
+        rs => abwesenheitMapping.opt(abwesenheit)(rs)
```

Comment:
Does this work without mapping the `vertrieb`? (can be, I just don't know.)

Reply:
yes, it does. I was also with the doubt